### DISCUSSION TITLE
Filtering events on dashboard

### DISCUSSION THREAD

**Discussion by En3pY** [wazuh/wazuh#25638] [General]
Not sure if this is the right place, eventually it will just not go further.

We've recently deployed a 4.9 version, and it appears to be working smoothly from data collection point of view, no major issues with it.

The dashboard looks very neat, but it has a really relevant downgrade when it comes to events drilling: in 4.8 version, each event row allowed you to zoom/filter out results for a given rule. In 4.9 this features is completely gone: clicking on an event row just shows the detail, and if you want to filter that out, you have to do it manually. As I said, it seems like a major downgrade from the previous features when hunting for events.

Is this going to be solved any time soon?

Apologies if I posted in the wrong place, but couldn't really find other ideas on where to ask for this.

**Comment by AT-SGAI-CSIC** [comment]
Hi, After update to 4.9 we have observed the same behaviour, for us this feature was fundamental in order to debug the alerts ... It's a bug?

**Reply by En3pY** [reply]
I think it's due to a major re-design of the interface and will probably be fixed.

But I found a workaround on this, even if it's much slower than before.

If you click on any of the records in threat hunting, the right pane opens. Click on "View surrounding documents" and another apge opens up. Here, in the list of events, you can click on the arrow to the right of the timestamp and it shows again all the elements where you can add the filters... it's much slower, but at least you can keep hunting for threats in the meantime.

![image](https://github.com/user-attachments/assets/be990c6f-4444-48f8-88d7-6dda4bacc42b)

![image](https://github.com/user-attachments/assets/048bd3a3-15e7-48f2-8ab0-007644cfbe0e)

**Reply by AT-SGAI-CSIC** [reply]
Ok, that's right I see that workaround but it is not practical. The advantage of the event part was precisely the ease of organizing the columns, in this new way it is very confusing ... but is it a bug?

**Reply by AravindSRaja** [reply]
You can still use the older discover page by toggling the New Discover (on/off).

![image](https://github.com/user-attachments/assets/e4bd6529-5644-4f92-a083-9af52bee6820)

**Reply by AT-SGAI-CSIC** [reply]
Ok, thanks, but this option it's only available in Discover ...
